common-service-core/src/main/java/org/zowe/apiml/zaas/ZaasResponse.java

```diff
@@ -17,8 +17,9 @@
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-public class ZosmfResponse {
+public class ZaasResponse {
```

Comment:
I think there would be more types of ZaasResponses, this one looks more like "ZaasTokenResponse"